### DISCUSSION TITLE
rocketmq 发送延时消息问题

### DISCUSSION THREAD

**Discussion by kecongcong** [apache/rocketmq#6902] [Q&A] [answered]
为什么rocketmq server是4.8.0版本，发送延时消息只能级别是1s和5s的消息，超过这个时间的都无法发送，感觉很奇怪

**Comment by kecongcong** [comment]
![image](https://github.com/apache/rocketmq/assets/29249422/5c35eeaa-0611-42c8-9317-90cd84000c39)
1和5可以发成功，其他的都不行

**Comment by kecongcong** [comment]
![image](https://github.com/apache/rocketmq/assets/29249422/cb46deea-940e-41ec-b4da-fca409f62779)
这些步骤都看过了，
![image](https://github.com/apache/rocketmq/assets/29249422/7a2b01fb-3562-4156-bea4-d188688be951)

**Comment by RongtongJin** [comment, marked as answer]
有一定可能性是触发了以下bug，可以打patch或升级版本进行修复，https://github.com/apache/rocketmq/pull/3358

**Reply by kecongcong** [reply]
感谢，升级完版本后可以触发了
